### PR TITLE
Added test suite on /test

### PR DESCRIPTION
**Pull request by mytja**
Hello, @alexmercerind and @raitonoberu !
I added a test suite to check if searching is working!
I also planned to add if downloads work, but this is too much currently!
I didn't make it ASYNC! I tried, but failed, so you can make it async!
Tell me what do you think!
Thank you!

### CODE REVIEW

File: build [hunk 1]
```diff
@@ -0,0 +1 @@
+gunicorn3 --bind 0.0.0.0:8000 main:app -w 4 -k uvicorn.workers.UvicornWorker
```
Comment (mytja) [inline build:1]: I used this command on my VM in the cloud!
You can execute it on ubuntu with ```source build``` or on Debian ```./build```

**Review by mytja — COMMENTED** [review]
And again, this is a full script to setup your server on non-heroku based server
You can run it like this on Ubuntu ```source setupvm``` or on Debian ```./setupvm```

File: setupvm [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+# Setup venv
+python3 -m venv venv
+source venv/bin/activate
+
+# Python PIP setup
+python3 --version
+python3 -m pip install -r requirements.txt
+
+# Run gunicorn script in background
+nohup build
```
Comment (mytja) [inline setupvm:27]: Nohup works on all Linux-es

Comment (mytja) [inline setupvm:27]: Nohup works on all linux-es

File: setupvm [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+
+# Gunicorn & Python & Nohup setup
+sudo apt-get install python3-venv
+sudo apt-get install python3-pip
+sudo apt-get install gunicorn3
+sudo apt-get install nohup
+
+# Setup venv
+python3 -m venv venv
+source venv/bin/activate
```
Comment (mytja) [inline setupvm:20]: Source as I think also works on all Linuxes, so it shouldn't be a problem!

**Comment by raitonoberu** [comment]
Great idea! The script that allows you to run on a Linux server is also useful. Let me know as soon as you're ready, and I'll make the tests asynchronous! 😇

**Comment by alexmercerind** [comment]
@mytja just a bit of advice... only commit & push once you are done with the changes. 😅

**Comment by mytja** [comment]
DONT MERGE YET!

**Comment by mytja** [comment]
You can merge everything now!

**Review by alexmercerind — APPROVED** [review]
I'm just approving. I don't really care apart from the fact that, you spent a lot of time on this.

**Comment by mytja** [comment]
Now we got a lot of failures, as it's supposed to be:
```
{
    "endtime": "Wed Dec 16 10:57:23 2020",
    "fail": true,
    "tracksearch": "Fail!",
    "albumsearch": "Fail!",
    "artistsearch": "Fail!",
    "trackdownload": "Fail!"
}
```

File: main.py [hunk 4]
```diff
@@ -18,7 +19,7 @@ def ReturnResponse(response):
 
 @app.get("/")
 async def hello():
-    return PlainTextResponse("service is running")
+    return ReturnResponse("service is running")
```
Comment (mytja) [inline main.py:22]: Here i just changed a little bit, to use what you built

File: main.py [hunk 5]
```diff
@@ -65,6 +66,73 @@ async def TrackDownload(track_id=None, album_id=None, track_name=None):
...
+        ifin =  "track_id" in response
+        if (response != None and ifin==True):
+            __musicsearchtest = "OK!"
+        else:
+            __musicsearchtest = "Fail!"
+    except:
+        __musicsearchtest = "Fail!"
+    
+    try:
+        response = await SearchYoutube("NCS", "album")
```
Comment (mytja) [inline main.py:85]: I made ti async! Don't know how, but I did it! ;-)

**Pull request merged**